### PR TITLE
Fdroid metadata

### PR DESCRIPTION
**Pull request by ShehabAhmed-Gui**
add F-droid metadata

### CODE REVIEW

File: metadata/en-US/changelogs/44000.txt
Comment (albexk) [inline metadata/en-US/changelogs/44000.txt:1]: It is necessary to delete this file. I don't think we will release version 4.4.0 on fdroid.

File: metadata/en-US/changelogs/45300.txt
Comment (albexk) [inline metadata/en-US/changelogs/45300.txt:1]: It is necessary to rename this file. The current versionCode is in `/CMakeLists.txt`: `set(APP_ANDROID_VERSION_CODE 52)`. Please also note this in the build metadata file. And versionName matches the version of the project (4.5.3.0).

File: metadata/en-US/changelogs/45300.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+What's Changed:
+* WireGuard obfuscation feature added
```
Comment (albexk) [inline metadata/en-US/changelogs/45300.txt]: The changelog needs to be replaced. Android changelog is different from the changelog in the release on github. The current android changelog is available from google play.

File: metadata/en-US/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Free open-source software to create a personal VPN on your server. Helps to access blocked content without revealing privacy even to VPN providers. Specify IP address, credentials for your server and Amnezia will configure it to connect via VPN automatically.
+
+We are not a VPN service, you don't have to connect to our servers and pay us anything. You free to use your own self-hosted VPS or you could purchase any VPS from any provider. To connect, use your own or purchase any VPS from any provider.
+
+Features
+
+• Connect using a config file, text code, by scanning a QR code, or setting up your own server
+• Select and set the protocols manually or use the "Wizard" function to configure them automatically based on the level of censorship in your country.
+• Supports the following VPN protocols on the server: WireGuard, AmneziaWG, OpenVPN, OpenVPN over Cloak, ShadowSocks and IKEv2.
```
Comment (albexk) [inline metadata/en-US/full_description.txt]: ShadowSocks and IKEv2 are not supported on Android, so it's better to remove them from this list. @pokamest, or is it better to completely copy the description from google play?

File: metadata/en-US/title.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+AmneziaVPN - Multi protocol VPN client
```
Comment (albexk) [inline metadata/en-US/title.txt]: Perhaps here it is better to take the name also from google play: just `Amnezia VPN`.

**Review by albexk — CHANGES_REQUESTED** [review]

File: metadata/en-US/full_description.txt (hunk 2 above)
Comment (pokamest) [reply metadata/en-US/full_description.txt]: Agree

**Review by albexk — APPROVED** [review]

**Pull request merged**